EVE-1146 ensure we are able to install the buildbot-worker with python 2

tcarmet opened the pull request.

tcarmet committed:
EVE-1146 ensure we are able to install the buildbot-worker with python 2

bert-e commented:
# Hello tcarmet,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/eve/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue EVE-1146 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `1.7.7`


Please check the `Fix Version/s` of EVE-1146, or the target
branch of this pull request.

cmeron-scality reviewed (commented):
LGTM, i'l let a chance for someone that knows Eve better to take a look first.
Ping me tomorrow if its still stuck :)

cmeron-scality reviewed (approved):
Approving then.

tcarmet committed:
EVE-1146 upgrading pip version to install worker dependencies

in order to fix issues with some OS unable to install
the buildbot-worker dependencies, we're upgrading pip version.
